variables.tf

```
@@ -73,6 +73,47 @@ variable "cloud_monitoring_instance_crn" {
   default     = null
 }
 
+##############################################################
+# CSPM
+##############################################################
+
+variable "cspm_enabled" {
+  description = "Enable Cloud Security Posture Management (CSPM) for the Workload Protection instance. This will create a trusted profile for the App Config instance and associate it with the Workload Protection instance."
```

Since we are removing `is_enterprise_account` and always granting the enterprise access, lets update this:
```suggestion
  description = "Enable Cloud Security Posture Management (CSPM) for the Workload Protection instance. This will create a trusted profile associated with the SCC Workload Protection instance that has viewer / reader access to the App Config service and viewer access to the Enterprise service."
```

---

make same change for DA